app/models/event.rb

```diff
@@ -10,16 +9,20 @@ class Event < ActiveRecord::Base
   validates :event_type_id, presence: true
   validates :date, presence: true
 
+  # Callbacks ##################################################################
+
+  before_save { self.name = name.titleize }
+
   # Scopes #####################################################################
 
-  scope :excusable, -> { where(self_submit_excuse: true) }
+  scope :excusable, -> { where("self_submit_excuse = :value AND date > :today",
```

Comment:
Style/MultilineBlockLayout: Block body expression is on the same line as the block start.